app/helpers/events_helper.rb

```diff
@@ -0,0 +1,9 @@
+module EventsHelper
+  def image_event(event)
+    image_tag event.image.url :medium if event.image.exists?
+  end
+
+  def created_by(event)
+    @users.find_by_id(event.user_id).name
+  end
```

Comment:
That helper is not needed
You can directly access to the event author with `event.user.name`